host/sgx/calls.c

```diff
@@ -365,6 +365,47 @@ static oe_result_t _handle_call_host_function(
     return result;
 }
 
+static const char* oe_ocall_str(oe_func_t ocall)
+{
+    switch (ocall)
```

Comment:
We can simplify here with a string array, e.g.:
```
    static const char* func_names[] = 
    {
        "OE_OCALL_CALL_HOST_FUNCTION",
        "OE_OCALL_GET_QE_TARGET_INFO",
        ...
    };
     if (ocall < __OE_FUNC_MAX)
       return func_names[ocall - OE_OCALL_BASE];
```
This way future OCALLs can be more easily added.

Reply:
Ok, will do. I’d deliberately copied what’s there for oe_result_t to be consistent but I agree a string array is more elegant.

Reply:
Done